components/form_inputs/autocompletes/AutocompleteFunctionality.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <v-container>
+    <h2>Functionality</h2>
+    <v-card>
+      <v-container>
+        <v-row align="center">
+          <v-col cols="40">
+            <v-autocomplete
+                    disabled
+                    dense
+                    v-model="values"
+                    :items="items"
+                    label="Disabled"
+            ></v-autocomplete>
+          </v-col>
+          <v-col cols="12">
+            <v-autocomplete
+                    clearable
+                    open-on-clear
+                    v-model="values"
+                    :items="items"
+                    dense
+                    label="Clearable, open on clear"
+            ></v-autocomplete>
+          </v-col>
+          <v-col cols="12">
+            <v-autocomplete
+                    auto-select-first
+                    v-model="values"
+                    :items="items"
+                    label="Autoselect first"
+                    dense

```

Comment:
In matrix only 'disabled', 'clearable' and 'open-on-clear' are mentioned.
Could you, please, clarify for what purpose you add this extra autocomplete?
It may confuse a little bit :)

Reply:
I think it's my mistake. There's feature _autoselect-first_ that is not included in our test site and i didn't add it to matrix and task description.